sops/european-level/GDI-SOP0003_1+mg-dac-recommendation-approval.md

```diff
@@ -33,12 +34,25 @@
 Find GDI SOPs common Glossary at the [**charter document**](https://github.com/GenomicDataInfrastructure/standard-operating-procedures/blob/main/docs/GDI-SOP_charter.md).
 
 For this SOP in particular, the source of truth for most definitions and terms is the [1+MG Data Access Governance for Research](https://docs.google.com/document/d/19HVf7SP2R_fCI-KugVStZR4yEAkV6vhvSbVd6a2ZviU).
+
 | Abbreviation | Description                                                                                               |
```

Comment:
In this list, it seems that you do not follow the alphabetical order. Is there any reason for that?

Reply:
The reason is likely that I didn't pay attention 😅 
Thanks for picking it up! I'll fix the glossaries. I tend to order them, and then add a new one and forget to reorder.